cns/middlewares/k8sSwiftV2_test_linux.go

```diff
@@ -69,8 +75,8 @@ func TestIPConfigsRequestHandlerWrapperSuccess(t *testing.T) {
 	happyReq.OrchestratorContext = b
 	resp, err := wrappedHandler(context.TODO(), happyReq)
 	assert.Equal(t, err, nil)
-	assert.Equal(t, resp.PodIPInfo[2].PodIPConfig.IPAddress, "192.168.0.1")
-	assert.Equal(t, resp.PodIPInfo[2].MacAddress, "00:00:00:00:00:00")
+	assert.Equal(t, resp.PodIPInfo[0].PodIPConfig.IPAddress, "192.168.0.1")
```

Comment:
why are we checking for the first PodIPInfo here instead of the third? Did we change the order of append for the PodIPInfos?

Reply:
Addressed.